manage

```diff
@@ -31,8 +31,8 @@ backup() {
   echo "Done (Deleting old backups (if any))"
 
   echo "Writing new backup files"
-  sudo tar -cspf $fakemount ./test/fakemount
-  sudo su -c "cd /var/lib/docker/volumes && tar -cspf $vols ./nca*"
+  sudo tar -cpf $fakemount ./test/fakemount
+  sudo su -c "cd /var/lib/docker/volumes && tar -cpf $vols ./nca*"
```

Comment:
Did y'all used to have sparse files that this had the `-s` option before? I think that was recently added in another PR earlier this work cycle, right? Or was it just accidentally committed earlier and never needed?

Reply:
You're thinking of `-S` (capital s). In this case it was one of the flags I messed with while trying to get the archives to preserve every aspect of the original filesystem, and which culminated in the hack where I forcibly change permissions at the end of the manage "restore" command.

Unfortunately I left in `-s` on accident, which is *an invalid argument* when creating an archive 🙄

I shouldn't have put the fix in this branch, though - I'm not sure how I managed that. Want me to pull it and stuff it in its own branch?

Reply:
Nah, it's fine in here. No worries. This comment thread is evidence it got review attention. Putting it in another branch now would just be extra work for formality's sake.